### ISSUE TITLE
Cover remains even after stopping playback

### ISSUE THREAD

**Issue by orickcorreia** [kalkih/mini-media-player#376]
I updated today to the latest version and noticed that the cover remains as a background even after it stops playing.

In the image above I transferred a reproduction of the Chromecast to the Google Mini and look what happened to the cover.
![image](https://user-images.githubusercontent.com/58919667/89345829-141ab100-d676-11ea-88c0-19c6966c9009.png)

**Comment by nagyrobi** [comment]
Start media palyback to see the cover.
In developer tools, look for your media player entity and see for the attribute `entity_picture`, what value does it show?
Stop media player. If you still see the cover, go back to the developer tools, select any other entity (a switch for example, to clear the webpage cache) and then select again your media player entity. Is the `entity_picture` still there, and if is, what's the value?

When media is stopped, `entity_picture` should be null.

**Comment by kalkih** [comment]
Yeah, check the attributes as @nagyrobi described, if the entity_picture is still set, the artwork will be displayed.
What browser are you running?
I have a chromecasts myself and they seem to clear artwork as expected.

**Comment by orickcorreia** [comment]
Stand-by
![image](https://user-images.githubusercontent.com/58919667/89491596-9d68db00-d77d-11ea-830d-c687ffcd2241.png)

Playing
![image](https://user-images.githubusercontent.com/58919667/89491622-a9ed3380-d77d-11ea-8d14-9236df114b5f.png)

Stoped
![image](https://user-images.githubusercontent.com/58919667/89491690-cee1a680-d77d-11ea-926d-5cb14ca56919.png)

I use some png backgrounds on my mmp. I noticed that the png image is defined as background-image in mmp__bg.

When I'm playing something, the cover of the song is replacing my default background. And when playback is stopped, the background image remains with the media artwork.

I noticed that if I remove the variable "--bg" my background disappears and the behavior of the artwork goes back to normal.

![image](https://user-images.githubusercontent.com/58919667/89492221-14eb3a00-d77f-11ea-9682-fe6e19126946.png)
